src/server/status/metrics.js

```diff
@@ -1,39 +1,71 @@
-import _ from 'lodash';
+import { get, set, isObject } from 'lodash';
 import { keysToSnakeCaseShallow } from '../../utils/case_conversion';
+import { getAllStats as cGroupStats } from './cgroup';
+
+let cGroupStatsAvailable = true;
 
 export function collectMetrics(kbnServer, server, config) {
-  server.plugins['even-better'].monitor.on('ops', function (event) {
-    kbnServer.metrics = getMetrics({ event, config });
+  server.plugins['even-better'].monitor.on('ops', event => {
+    getMetrics(event, config, server).then(data => { kbnServer.metrics = data; });
   });
 }
 
-export function getMetrics({ event, config }) {
+export async function getMetrics(event, config, server) {
   const port = config.get('server.port');
   const timestamp = new Date().toISOString();
```

Comment:
Having it right at the ops event is probably best as it's actually involved in aggregations (request count, etc). Capturing cgroups is also a very quick operation in that it's reading most of the files in parallel and they aren't actually on disk. We should revisit when we add CPU percentage.